src/imt/side.rs

```diff
@@ -10,6 +10,52 @@ pub struct Buy;
 #[derive(Debug, Clone, Copy)]
 pub struct Sell;
 
+/// Type markers for Buy and Sell sides
+#[derive(Debug, Clone, Copy)]
+pub struct Aggressive;
+#[derive(Debug, Clone, Copy)]
+pub struct Passive;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum MatchType {
+    Aggressive,
+    Passive,
+}
+
+/// Order match type marker trait with compile-time constants
+pub trait OrderMatchType: 'static + Copy + Send + Sync + Clone + Copy + Debug {
```

Comment:
```suggestion
pub trait OrderMatchType: 'static + Copy + Send + Sync + Clone + Debug {
```

Contains double copy trait

Reply:
fixed